common/utils/dateHelpers.ts

```diff
@@ -264,6 +264,16 @@ export function formatFullDate(date: Date | number, withYear: boolean = false):
     return `${dayOfWeek}, ${day} ${month}`;
 }
 
+export function formatDateDayMonthYear(date: Date | number): string {
+    const d = getDate(date);
```

Comment:
is there a clearer name we can use than d?

Reply:
I agree a one char name is not ideal but it was done to match the style of the other functions in this file. I think it should be fine as these are only small helpers